bin/critterget.py

```diff
@@ -156,16 +156,26 @@ def getCrashSummary(appId, appName):
     mystime = scopetime()
 
     crashattrs = "hash,lastOccurred,sessionCount,uniqueSessionCount,reason,status,displayReason,name"
-    crashdata = apicall("app/%s/crash/summaries" % appId, "lastOccurredStart=%s" % mystime)
-    CrashDict = {}
-    for x,y in enumerate(crashdata):
-        printstring = u'{} MessageType="CrashSummary" appId={} appName="{}" '.format(DATETIME_OF_RUN, appId, appName)
-        slist = crashattrs.split(",")
-        for atname in slist:
-            printstring += u'{}="{}" '.format(atname, crashdata[x][atname])
-            if atname == "hash" : CrashDict[crashdata[x][atname]]= appName
-        print printstring
 
+    http_code = None
+    retry = 1
+    while http_code != 200:
+        http_code, crashdata = apicall_with_response_code("app/%s/crash/summaries" % appId, "lastOccurredStart=%s" % mystime)
+        retry += 1
+        if retry > MAX_RETRY:
+            break
+
+    CrashDict = {}
+    if http_code != 200:
+        print u'{} MessageType="CrashSummary" appId={} appName="{}" ERROR COLLECTING CRASH SUMMARIES'.format(DATETIME_OF_RUN, appId, appName)
```

Comment:
I recommend adding more detail to this log message.  Such as the http_code you got (so you can confirm it's a 500 and not something else), and the value of retry. (confirming that your retry code was exercised)  Unless this is stuff that customers would see in Splunk and we wouldn't want to show.